larq/quantizers.py

```diff
@@ -554,3 +554,10 @@ def get(identifier):
     raise ValueError(
         f"Could not interpret quantization function identifier: {identifier}"
     )
+
+
+def get_kernel_quantizer(identifier):
```

Comment:
Do we want to make this a private method? (And add a docstring?)

Reply:
I added a docstring.

I don't think we should make it a private method since we use it in `layers_base.py`  which is outside of this file.